taskcluster/async/asyncclient.py

```diff
@@ -50,10 +53,16 @@ class AsyncBaseClient(BaseClient):
     help with this.
     """
 
+    def __init__(self, *args, **kwargs):
+        super(AsyncBaseClient, self).__init__(*args, **kwargs)
+        self._implicit = False
+        if self.session is None:
+            self._implicit = True
```

Comment:
can we we make this name more explicit as to what is implict?  Maybe `_implicitSession`?

Reply:
Done.